docs/csharp/programming-guide/concepts/covariance-contravariance/variance-in-generic-interfaces.md

```diff
@@ -6,7 +6,9 @@ ms.assetid: 4828a8f9-48c0-4128-9749-7fcd6bf19a06
 
```

Comment:
line 3 contains ms.date, the date this topic was last updated. Please change it to today's date.

Reply:
Yep, will do.